controller/src/main/java/io/pravega/controller/task/KeyValueTable/TableMetadataTasks.java

```diff
@@ -264,15 +265,15 @@ public String retrieveDelegationToken() {
     }
 
     public CompletableFuture<Void> createNewSegments(String scope, String kvt,
-                                                      List<Long> segmentIds, long requestId) {
+                                                      List<Long> segmentIds, int keyLength, long requestId) {
```

Comment:
It may be good to create another overloaded `createNewSegments()` method that creates all segments with keyLength=0 (_does not take `keylength` as a parameter and invokes this method_), so existing non-KVT createNewSegments() calls don't need to change.

Reply:
Oh yes, didn't realize this is a method in TableMetadataTasks.